### ISSUE TITLE
The graph data persists after titan.sh clean because of data being written to bin/db/cassandra

### ISSUE THREAD

**Issue by denisekgosnell** [thinkaurelius/titan#825]
The strange behavior I noticed: there were cases when I would stop titan and remove all data with titan.sh clean. After restarting titan, the graph would still be accessible and was not removed.

The reasons this is happening on my system:
1) titan.sh clean looks in `titan/log` and `titan/db/` to remove all data from the Cassandra and elastic search backends
2) If you start titan from within titan/bin/, then the Cassandra data is mapped to be stored in:
titan/bin/db/cassandra/ whereas the elastic search data is mapped to titan/db 
--> As such, when you run titan clean, the data is not properly removed.

How I can avoid this at the moment: I can manually remove all cassandra data and be sure to start titan in the proper directory.

A Proposed Fix: the titan start and associated Cassandra launching scripts could be sure to map the path of the Cassandra data to be in titan/db/cassandra regardless of the absolute path from which titan was started (Eg: if it is started from within bin)

**Comment by dalaro** [comment]
@denisekgosnell which version of Titan is this (please include the full name of the zipfile if the problem appeared while using one of the zipfile downloads)?

**Comment by denisekgosnell** [comment]
@dalaro 
from http://s3.thinkaurelius.com/downloads/titan/
we are using: titan-0.5.0-hadoop2.zip

**Comment by dalaro** [comment]
Thank you, I reproduced the bug with your report.  I then modified cassandra.in.sh, the shell fragment evaluated shortly before the scripts fork the Cassandra daemon, to change the working directory that which contains the `db` directory (b2ad7e883d873b1e8425aedc31cefbf82aacb60c).  Now the Cassandra daemon starts with the same working dir regardless of the working dir from which `titan.sh` was invoked.

I also noted this in the tentative upgrade instructions for the upcoming 0.5.2 release (e8e7201de14cfdbb4c3c90b20419531b6d663cb9).
